docs/modules/ROOT/pages/java.adoc

```diff
@@ -827,10 +801,9 @@ Gateway
 https://developer.couchbase.com/documentation/mobile/1.5/installation/sync-gateway/index.html[installation
```

Comment:
Correct link for 2.0 Sync Gateway should be?
https://developer.couchbase.com/documentation/mobile/2.0/installation/sync-gateway/index.html [installation guide]